backend/typescript/services/implementations/campService.ts

```diff
@@ -537,24 +574,26 @@ class CampService implements ICampService {
 
         const campFee = newCamp.fee;
 
-        const dropoffPriceObject = await createStripePrice(
-          dropoffProductId,
-          dropoffFee * 100,
-        );
+        if (pickupFee && dropoffFee) {
+          const dropoffPriceObject = await createStripePrice(
```

Comment:
are these objects necessary? looks like we only need the IDs here

Reply:
I didn't want to refactor the stripe API code in this PR since it's already wayyyyyy too big. I think we can revisit these improvements in future PRs?